submissions/dafen173/js-dom/script.js

```diff
@@ -0,0 +1,171 @@
+
+// Menu button
+const menubtn = document.getElementById('menubtn');
+const dropchild = document.getElementById('dropchild');
+menubtn.addEventListener("click", function(){
+  dropchild.classList.toggle('open');
+});  
+
+
+let inst = [{id:'section1', name:'6500 ANSI Lm на улице', description:'Работает проектор типа HITACHI CP-WX8265 3LCD  Lamp  1280х800   WXGA 6500lm  3600уе на экран шириной 3 м.', img:'img/first.png', img2:'img/0697.png'},
```

Comment:
Try to use a more readable format of the object with data. I mean every key of the object at the new line.

Reply:
done

https://github.com/dafen173/kottans_frontend/blob/master/task_js_dom/script.js

ok?

Reply:
Use Prettier for automatically formatting your code.